votor-messages/src/consensus_message.rs

```diff
@@ -1,4 +1,4 @@
-//! Put Alpenglow consensus messages here so all clients can agree on the format.
```

Comment:
I think we should either stick with the previous comment (my preference) or lean all the way in and call this `bls_message.rs`